source/agora/common/TransactionPool.d

```diff
@@ -102,7 +102,7 @@ public class TransactionPool
             log.info("Loading database from: {}", db_path);
 
         // note: can fail. we may want to just recover txes from the network instead.
```

Comment:
Oh. I think we should examine this again. Getting the tx's other nodes' pools would be a useful feature.